bin/openmetadata.sh

```diff
@@ -102,6 +103,28 @@ function getPID {
    return 0
 }
 
+# Check if the mysql migrations are up to date
+function validateDbMigrations {
```

Comment:
@pmbrull do we need this function, give you have it in CatalogApplication?

Reply:
Removed 🙏